src/main.js

```diff
@@ -88,23 +88,26 @@ $(document).ready(() => {
     async function optionsChanged(event, changes) {
       let reload = false;
 
-      Object.keys(changes).forEach((storeKey) => {
-        const [oldValue, newValue] = changes[storeKey];
-
-        switch (storeKey) {
-          case STORE.TOKEN:
-          case STORE.LOADALL:
-          case STORE.ICONS:
-          case STORE.PR:
-            reload = true;
-            break;
-          case STORE.HOVEROPEN:
-            handleHoverOpenOption(newValue);
-            break;
-          case STORE.HOTKEYS:
-            setHotkeys(newValue, oldValue);
-            break;
+      const reactions = {
+        [STORE.TOKEN] () {},
```

Comment:
This mighe be a bug. Token / LoadAll / Icons / Pull Requests should all trigger reload. But look like with this logic, it doesn't do that.

Reply:
Oh yes it is, do you think i should remove the lines 92, 93, 94?

Reply:
or just use the old switch case logic would be easier to read

Reply:
Agreed

Reply:
Yep agreed